src/wp-admin/includes/upgrade.php

```diff
@@ -2135,23 +2438,23 @@ function __get_option( $setting ) { // phpcs:ignore WordPress.NamingConventions.
  * @param string $content The content to modify.
  * @return string The de-slashed content.
  */
-function deslash($content) {
+function deslash( $content ) {
 	// Note: \\\ inside a regex denotes a single backslash.
 
 	/*
 	 * Replace one or more backslashes followed by a single quote with
 	 * a single quote.
 	 */
-	$content = preg_replace("/\\\+'/", "'", $content);
+	$content = preg_replace( "/\\\+'/", "'", $content );
 
 	/*
 	 * Replace one or more backslashes followed by a double quote with
 	 * a double quote.
 	 */
-	$content = preg_replace('/\\\+"/', '"', $content);
+	$content = preg_replace( '/\\\+"/', '"', $content );
 
 	// Replace one or more backslashes with one backslash.
-	$content = preg_replace("/\\\+/", "\\", $content);
+	$content = preg_replace( '/\\\+/', '\\', $content );
```

Comment:
The slashes aren't the same in single quotes.

Reply:
Seems this is the same (single quotes) upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/upgrade.php#L2705

Reply:
It needs a little test to make sure, because the comment says what should happen, and that doesn't match what is in single quotes.

Reply:
I was misreading it. My mistake.